src/plugins/text-mode/index.ts

```diff
@@ -40,12 +40,19 @@ export default class TextMode extends PluginController {
     this.cc.updateViews();
   }
 
+  abortScrollListenerController = new AbortController();
```

Comment:
Ooh I haven't seen AbortController before. Nicer way to abort the listener than the old API